public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
     "lng": -2.5934541,
     "id": "94e94f50-1d30-49f0-8959-b2a799ba3423",
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
+    "address": "Park Lane\nCheshire East\nPoyton\nSK12 1GP",
+    "phone": null,
+    "hours": null,
+    "lat": 53.3494644,
+    "lng": -2.1180696,
+    "id": "dc4e84b0-ab6d-44a3-8090-d633b5ed5313",
+    "booking_location_id": "df38d9b9-e9a9-4bf0-b56b-4309965b815c"
+  },
+  {
+    "title": "Peterlee Macmillan Information Centre",
+    "address": "20 The Upper Chare\nPeterlee\nSR8 1BW",
+    "phone": null,
+    "hours": null,
+    "lat": 54.7589267,
+    "lng": -1.3343109,
+    "id": "5b061a57-4095-499e-a63b-26f4585a03ea",
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "Bishop Auckland Hospital",
+    "address": "Cockton Hill Road\nBishop Auckland\nDurham\nDL14 6AD",
+    "phone": null,
+    "hours": null,
+    "lat": 54.6532811,
+    "lng": -1.6778693,
+    "id": "3e5f4f69-56cc-4ff1-bda5-d61ef1592693",
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "University Hospital North Durham Hospital",
```

Comment:
"Durham" and "University Hospital North Durham Hospital" as the first line of the address. If there are multiple locations in Durham we could set the title to "Durham (University Hospital)"